django_project/social_users/views.py

```diff
@@ -97,39 +110,57 @@ def save_profile(backend, user, response, *args, **kwargs):
     return {'user': user}
 
 
-def user_updates(user, date):
+def user_updates(user, date, osm_user):
     updates = []
-    # from locality archive
-    ids = (
-        LocalityArchive.objects
-        .filter(changeset__social_user=user).filter(changeset__created__lt=date)
-        .order_by('-changeset__created')
-        .values('changeset', 'object_id')
-        .annotate(id=Min('id')).values('id')
-    )
-    updates_temp = (
-        LocalityArchive.objects
-        .filter(changeset__social_user=user).filter(changeset__created__lt=date)
-        .filter(id__in=ids)
-        .order_by('-changeset__created')
-        .values('changeset', 'changeset__created', 'changeset__social_user__username', 'version')
-        .annotate(edit_count=Count('changeset'), locality_id=Max('object_id'))[:10]
-    )
     changesets = []
-    for update in updates_temp:
-        changesets.append(update['changeset'])
-        updates.append(get_update_detail(update))
-
-    # get from locality if not in Locality Archive yet
-    updates_temp = (
-        Locality.objects
-        .filter(changeset__social_user=user).exclude(changeset__in=changesets)
-        .order_by('-changeset__created')
-        .values('changeset', 'changeset__created', 'changeset__social_user__username', 'version')
-        .annotate(edit_count=Count('changeset'), locality_id=Max('id'))[:10]
-    )
-    for update in updates_temp:
-        updates.append(get_update_detail(update))
+
+    if not osm_user:
```

Comment:
i think we don't need this section, all of user is using osm data (use HS username to check on docker osm data)